view/frontend/templates/menu/node_type/custom_url.phtml

```diff
@@ -13,9 +14,16 @@ if ($block->getIsViewAllLink()) {
         $dataAttribute = ' data-menu="menu-' . $block->getId() . '"';
     }
 }
+
+if($block->getTarget()) {
+    $target = 'target="__blank"';
```

Comment:
Are you sure you want to force `target="__blank"` for custom links?

Reply:
we are forcing target="__blank if the  "Node target blank" option is checked http://prntscr.com/hpixk7